java-servlet-json/src/test/java/com/hmkcode/ArticleMapperTest.java

```diff
@@ -0,0 +1,86 @@
+package com.hmkcode;/**
+ * Author: wge
+ * Date: 22/11/2013
+ * Time: 18:09
+ */
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.hmkcode.vo.Article;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ArticleMapperTest
+{
+    private static final java.util.logging.Logger log = java.util.logging.Logger.getLogger(ArticleMapperTest.class.getName());
+
+
+    @Test
+    public void testValidJson() throws Exception
+    {
+        String json = readlineFromResourceFile("/valid.json");
+
+        ObjectMapper mapper = new ObjectMapper();
+       	Article article = mapper.readValue(json, Article.class);
+
+        assert(article.url.equals("news.bbc.co.uk"));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/maratmingazovr/Java/file/2724754916/issues/source?bid=3348317&fileBranchId=3348331#l31)